scripts/xtensa-build-zephyr.py

```diff
@@ -1169,6 +1170,32 @@ def reproducible_checksum(platform, ri_file):
 	chk256 = sof_ri_info.EraseVariables(ri_file, parsed_ri, west_top / repro_output)
 	print('sha256sum {0}\n{1} {0}'.format(repro_output, chk256))
 
+def create_firmware_bundle():
+	if shutil.which('tar') is None:
+		print("tar tool not found, not creating firmware tarball")
```

Comment:
Sorry I'm seeing this only now. Better to just use `import tarfile`

Reply:
This was semi-intentional to not support the feature on Windows as producing Linux /lib/firmware binaries is more of a Linux problem. But alas, why not right, so updated to use tarfile module. Might even work on Windows.